packages/wallet-sdk/src/CoinbaseWalletProvider.ts

```diff
@@ -44,6 +44,7 @@ export class CoinbaseWalletProvider extends EventEmitter implements ProviderInte
     // eth_requestAccounts
     handshake: async (_: RequestArguments): Promise<AddressString[]> => {
       if (this.connected) {
+        this.emit('connect', { chainId: prepend0x(this.chain.id.toString(16)) });
```

Comment:
i thought we didn't emit `connect` for already connected case?

Reply:
the playground testdapp is one example

it loses connected state on reload, and it needs this connect message to enable the rest of functionalities